pkg/log/log_test.go

```diff
@@ -22,13 +22,11 @@ import (
 )
 
 func ExampleLoggerFuncs() {
-	logger := log.Logger{}
-
 	xdsLogger := LoggerFuncs{
-		DebugFunc: logger.Printf,
-		InfoFunc:  logger.Printf,
-		WarnFunc:  logger.Printf,
-		ErrorFunc: logger.Printf,
+		DebugFunc: log.Printf,
+		InfoFunc:  log.Printf,
+		WarnFunc:  log.Printf,
+		ErrorFunc: log.Printf,
```

Comment:
Why do we need this change? Can we make a separate PR?

Reply:
I think the linter was complaining about this? Honestly can't remember haha